bin/bootstrax

```diff
@@ -1620,8 +1626,10 @@ def cleanup_db():
     abandon_queries = [
         ({'tags': {
             '$elemMatch': {
-                'name': 'bad'}}},
-         "Run has a 'bad' tag"),
+                'name': 'abandon'}},
+             'start': {'$gt': now(-timeouts['abandoning_allowed'])}
```

Comment:
We could redo this query as `{'tags.name': 'abandon', 'start': {'$gt': now(-timeouts['abandoning_allowed'])}}`. Also, do we want to consider the time when the tag was added?

Reply:
> Also, do we want to consider the time when the tag was added?

Not per se, usually one of the older ebs catches it within 10 s